swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+from swap_meet.item import Item
```

Comment:
Do we need this import? Are we using the literal name `Item` anywhere in this file? Does Python need us to import a module in order to work with an object passed in as a function parameter?